CHANGELOG.md

```diff
@@ -32,6 +35,13 @@
   updates to existing configurations.
 - `PATCH /network-interfaces/{id}` only allowed post-boot. Use `PUT` for
   pre-boot updates to existing configurations.
+- MMDS failure response for retrieval of unsupported JSON value types
```

Comment:
Changed returned status code from 500 to `501 Not Implemented` for queries on the MMDS endpoint in IMDS format where the value is unsupported.